src/Authentication/Services/SystemUserService.cs

```diff
@@ -300,5 +304,14 @@ public async Task<Result<SystemUser>> CreateAndDelegateSystemUser(string partyId
         {
             return await _repository.GetSystemUserByExternalRequestId(externalRequestId);
         }
+
+        /// <inheritdoc/>
+        public async Task<Result<Page<SystemUserRegisterDTO, long>>> GetAllSystemUsers(long continueFrom, CancellationToken cancellationToken)
+        {
+            List<SystemUserRegisterDTO>? theList = await _repository.GetAllSystemUsers(continueFrom, _paginationSize, cancellationToken);
+            theList ??= [];
+
+            return Page.Create(theList, _paginationSize, static theList => theList.SequenceNo);
```

Comment:
I *think* this might be paginating wrongly, but I can't say entirely for sure just based on this code. A `Stream` has different pagination semantics from normal pagination.